rollout/trafficrouting/istio/istio.go

```diff
@@ -26,8 +26,8 @@ const Type = "Istio"
 // NewReconciler returns a reconciler struct that brings the Virtual Service into the desired state
 func NewReconciler(r *v1alpha1.Rollout, client dynamic.Interface, recorder record.EventRecorder, virtualServiceLister, destinationRuleLister dynamiclister.Lister) *Reconciler {
 	return &Reconciler{
-		rollout: r,
-		log:     logutil.WithRollout(r),
+		rollout:               r,
+		log:                   logutil.WithRollout(r),
```

Comment:
missing go fmt?

Reply:
Sorry I forgot to run `go fmt` before pushing the changes. Will run and update the commit. Thanks!